modules/virt-configuring-cluster-real-time.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-cluster-realtime-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-cluster-real-time_{context}"]
+= Configuring a cluster for real-time workloads
+
+You can configure an {product-title} cluster to run real-time workloads.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Node Tuning Operator.
+
+.Procedure
+
+. Label a subset of the compute nodes with a custom role; for example, `worker-realtime`:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-realtime=""
+----
++
+[NOTE]
+====
+You must use the default `master` role for {sno} clusters.
+====
+
+. Create a new `MachineConfigPool` manifest that contains the `worker-realtime` label in the `spec.machineConfigSelector` object:
++
+.Example `MachineConfigPool` manifest
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-realtime
+  labels:
+    machineconfiguration.openshift.io/role: worker-realtime
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values:
+          - worker
+          - worker-realtime
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-realtime: "" 
+----
++
+[NOTE]
+====
+The above example manifest does not apply to {sno} clusters.
```

Comment:
@orelmisan @RamLavi 
I'm not sure if this is true (even though I commented about it in my previous review), because now I see that this is the same MCP I applied successfully (i.e. it ended with successful checkups) on my compact cluster.
WDYT?